src/main/java/uk/gov/justice/hmpps/prison/api/model/NewCaseNote.java

```diff
@@ -42,7 +43,7 @@
 
     private LocalDateTime occurrenceDateTime;
 
-    @Size(max = 4000)
+    @MaximumTextSize
```

Comment:
Seems like quite a generic name for a very specific rule - would it be better to give it a `max` parameter? It could then be dropped in as a replacement for `@Size` in other places if needs be...

Reply:
Not sure what swagger looks like for this but does it still document the maximum size of 4000?

Reply:
The purpose of the validation is to stop Oracle from throwing exceptions when the text size exceeds 4000 bytes, which is the max varchar size for Oracle. It is an absolute limit when saving data, hence the specific name.

Reply:
I thought 4000 chars was just the column size - didn't realise 4000 was a hard limit!  In which case it makes perfect sense to me.